rewriter/src/test/java/com/facebook/coresql/rewriter/TestOrderByRewriter.java

```diff
@@ -70,18 +73,22 @@
 
     private void assertOrderByPatternIsMatchedOrUnmatched(String sql, boolean isMatched)
     {
+        AstNode ast = parseStatement(sql);
+        assertNotNull(ast);
         Rewriter rewriter = new OrderByRewriter();
         if (isMatched) {
-            assertTrue(rewriter.rewritePatternIsPresent(sql));
+            assertTrue(rewriter.hasRewritePattern(ast));
```

Comment:
We don't need this tests either. Just test for non-empty result

Reply:
Done